site_root/input/resources/examples/Task-example-research-study-task-id.json

```diff
@@ -0,0 +1,34 @@
+{
+  "resourceType": "Task",
+  "id": "example-research-study-task-id",
+  "status": "accepted",
```

Comment:
Per http://hl7.org/fhir/R4/valueset-task-status.html, `"completed"` maybe since, upon the receipt of a sequencing manifest, the task itself has been already finished?

Reply:
resolved - implemented.